Dockerfile.venv

```diff
@@ -0,0 +1,14 @@
+FROM elifesciences/python:b22fbad8f66100d88cb4bd7c7d092b376a9e09bc
```

Comment:
could also be named `.pip`, but in the end both `pip` and `virtualenv` are needed here